.github/workflows/create-release-draft.yml

```diff
@@ -148,6 +157,42 @@ jobs:
             ${{ env.GENESIS_RELEASE }}-genesis-state
             ${{ env.GENESIS_RELEASE }}-genesis-wasm
 
+  ## Build the enclave and package config files
+  generate_package_and_artifact:
```

Comment:
maybe something like `build-enclave` would be more appropriate for the job name

Reply:
Done ✅